federatedscope/core/data/base_data.py

```diff
@@ -149,9 +149,9 @@ class ClientData(dict):
     """
     def __init__(self, client_cfg, train=None, val=None, test=None, **kwargs):
         self.client_cfg = None
-        self.train = train
-        self.val = val
-        self.test = test
+        self.train_data = train
```

Comment:
I don't think these changes are necessary as the names of the corresponding arguments of this constructor have not been changed. It would be better to provide a docstring for this constructor to elaborate on the arguments.